documentation/polymer-templates/tutorial-template-model-encoders.asciidoc

```diff
@@ -1,11 +1,15 @@
 ---
 title: PolymerTemplate, Using Model Encoders
-order: 11
+order: 16
 layout: page
 ---
 
 = PolymerTemplate, Using Model Encoders
 
+*Usage of polymer based templates is deprecated since Vaadin 18 and we recommend using Lit based templates instead.*
```

Comment:
- “Polymer” should be capitalised
- Better break onto two separate sentences
- I guess LitElement should be mentioned with a complete name, otherwise it is confusing, as there are also `lit-html` templates

```suggestion
*Usage of Polymer based templates is deprecated since Vaadin 18. It is recommended to use LitElement based templates instead.*
```